sky/services/platform/path_provider.mojom

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module flutter.platform;
+
+/// Returns commonly used locations on the filesystem.
+interface PathProvider {
+  /// Path to the read-only directory that contains the assets packaged with
+  /// the application bundle.
+  ///
+  /// Example(s):
+  ///    iOS: The application bundle directory.
+  AssetsDirectory() => (string path);
```

Comment:
Yeah, we shouldn't expose this.  We want to be able to serve assets out of the zip file or transcode to some other format.